docs/src/main/asciidoc/vertx.adoc

```diff
@@ -47,6 +47,44 @@ Otherwise, you can manually add this to the dependencies section of your `pom.xm
 </dependency>
 ----
 
+== Native Transport
+
+Vert.x is capable of using https://netty.io/wiki/native-transports.html[Netty's native transports] which offers
+performance improvements on certain platforms. To enable them you must include one or both of the following dependencies:
```

Comment:
How to know when to include one or both? Aren't these dependencies OS specific?

Reply:
The referenced documentation https://netty.io/wiki/native-transports.html is where you get that information. But it won't break if you include both of them. Netty is smart enough to fallback to a non native transport. Usually projects include both so they'll be platform agnostic.

Reply:
To be clear. If you include both netty is smart enough to know which one it should use (or non tall) depending on the platform.

Reply:
Gotcha. Thanks!

Reply:
I improved the documentation to make it clearer.